recipe/meta.yaml

```
@@ -31,14 +33,18 @@ test:
   commands:
     - pip check
     - pip install -e ".[test]"
-    - python -m pytest -vv
+    - python -m pytest -k "not (test_run or test_normalize_cmd)" -vv  # [win]
+    - python -m pytest -vv  # [not win]
   requires:
     - pip
+    - cryptography =39.0.1
 
 about:
   home: https://pypi.org/project/hatch-jupyter-builder/
   summary: A hatch plugin to help build Jupyter packages
-  dev_url: https://github.com/blink0173/hatch-jupyter
+  description: A build hook plugin for Hatch that adds a build step for use with Jupyter packages.
+  dev_url: https://github.com/jupyterlab/hatch-jupyter-builder
+  doc_url: https://hatch-jupyter-builder.readthedocs.io/en/latest/
```

```suggestion
  doc_url: https://hatch-jupyter-builder.readthedocs.io
```
If we omit en/latest/, non-english readers might get a version in their language.

---

Thanks for catching that. Looks like we have around 200 instances of that pattern, wonder if we might want a lint rule to help catch that.

---

Good idea, I opened one here: https://github.com/anaconda-distribution/anaconda-linter/issues/223